plugins/flashrom/fu-plugin-flashrom.c

```diff
@@ -183,11 +271,39 @@ fu_plugin_flashrom_startup(FuPlugin *plugin, GError **error)
 	return TRUE;
 }
 
+static gboolean
+fu_plugin_flashrom_unlock(FuPlugin *self, FuDevice *device, GError **error)
+{
+	FuFlashromDevice *flashrom_device = FU_FLASHROM_DEVICE(device);
+
+	if (fu_flashrom_device_get_region(flashrom_device) == FU_IFD_REGION_ME &&
+	    fu_flashrom_device_has_fn_m_me_unlock(flashrom_device)) {
+		g_set_error_literal(error,
+				    FWUPD_ERROR,
+				    FWUPD_ERROR_NOT_SUPPORTED,
+				    "\n"
+				    "ME region on Tuxedo laptops should be unlocked manually:\n"
+				    " 1. Power off your laptop\n"
+				    " 2. Press and keep holding Fn + M during the next step\n"
+				    " 3. Press power on button");
```

Comment:
I suspect @hughsie won't be a fan of hardcoded error strings, this means translators won't be able to help in translations.  But this is a pretty advanced use case.  Maybe it is reasonable.

If the strings stay, presumably you don't want to include the words "Tuxedo" or "laptop" to future proof the string.  That way other models may support it, and other form factors may support it.  You can move tuxdeo and laptop into a comment near by.

Reply:
I see only one use of translatable strings in plugins:
https://github.com/fwupd/fwupd/blob/7ec0f11e6925fd59f3a1a8f177b90de54f40a300/plugins/uefi-capsule/fu-plugin-uefi-capsule.c#L447
And it seems to not be used... I'll add `_(` and `TRANSLATORS:` comment as it seems to be supported in plugins.

Another thing is that verbose instructions are given via an error message, which is sort of a misuse, but I didn't see another way of doing it.

Reply:
Na, no translations in plugins please. Ideally it'd be an enumerated fwupdrequest type, but that's hard for this kind of thing.